data_scrapers/alameda_county.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+import requests
+import urllib.request
+from bs4 import BeautifulSoup
+import json
+import pandas as pd
+from typing import List, Dict
+from datetime import datetime, timedelta, timezone
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+import re
+import os
+
+# Note that we are using numbers for all of Alameda County, including Berkeley
+# Running this scraper requires a Firefox webdriver. The macos Firefox driver, geckodriver, is stored in ./env/bin
+# This is the open data landing page: https://data.acgov.org/search?source=alameda%20county%20public%20health%20department&tags=covid-19
+    # Getting strted with the ArcGIS REST API: https://developers.arcgis.com/rest/services-reference/get-started-with-the-services-directory.htm
+    # Services reference for Layer: https://developers.arcgis.com/rest/services-reference/layer-feature-service-.htm
+
+# URLs and API endpoints:
+landing_page = "https://data.acgov.org/search?source=alameda%20county%20public%20health%20department&tags=covid-19"
+cases_deaths = 'https://opendata.arcgis.com/datasets/7ea4fd9b8a1040a7b3815f2e0b5f92ba_0/FeatureServer/0/query'
+demographics = 'https://opendata.arcgis.com/datasets/f218564293f9400a8296558e9325f265_0/FeatureServer/0/query'
+cases_meta = 'https://services3.arcgis.com/1iDJcsklY3l3KIjE/arcgis/rest/services/AC_dates/FeatureServer/0?f=json'
+demographics_meta = 'https://services3.arcgis.com/1iDJcsklY3l3KIjE/arcgis/rest/services/AC_geography/FeatureServer/0?f=json'
+dashboards = ['https://ac-hcsa.maps.arcgis.com/apps/opsdashboard/index.html#/1e0ac4385cbe4cc1bffe2cf7f8e7f0d9',
+              'https://ac-hcsa.maps.arcgis.com/apps/opsdashboard/index.html#/332a092bbc3641bd9ec8373e7c7b5b3d']
+
+# Load data model template into the 'out' dictionary. 'out' will be global to all methods.
+with open('./data_scrapers/_data_model.json') as template:
+    out = json.load(template)
+
+def get_county() -> Dict:
+    """Main method for populating the 'out' dictionary"""
+    # populate dataset headers
+    out["name"] = "Alameda County"
+    out["source_url"] = landing_page
+    out["meta_from_source"] = get_notes()
+
+    # fetch cases metadata, to get the timestamp
+    response = requests.get(cases_meta)
+    response.raise_for_status()
+    cases_header = json.loads(response.text)
```

Comment:
I've changed the json loading to use response.json(). No reason for the change - I'm not familiar with either library and the inconsistencies are probably because I'm referencing examples from different sources.